bin/deliver_poste_messages

```diff
@@ -270,18 +271,21 @@ def main()
     until queue.empty?
       next unless delivery = queue.pop(true) rescue nil
 
+      sent_at = DateTime.now
+
       begin
         deliverer.send delivery
-      rescue Net::SMTPSyntaxError => e
-        puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
-      rescue Net::SMTPFatalError => e
+      rescue Net::SMTPSyntaxError, Net::SMTPFatalError => e
         puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
```

Comment:
should we also set sent_at to 0 in this case?
(I know that's the same as existing behavior but if you're defining a new meaning of sent_at=0 to mean "not sent but not going to be sent", which I think is a good idea, we should probably change the way we are canceling messages now)